yarn.lock

```diff
@@ -23,7 +23,7 @@
   resolved "https://registry.npmjs.org/@babel/compat-data/-/compat-data-7.25.2.tgz"
   integrity sha512-bYcppcpKBvX4znYaPEeFau03bp89ShqNMLs+rmdptMw+heSZh9+z84d2YG+K7cYLbWwzdjtDoW/uqZmPjulClQ==
 
-"@babel/core@^7.0.0", "@babel/core@^7.0.0-0", "@babel/core@^7.0.0-0 || ^8.0.0-0 <8.0.0", "@babel/core@^7.1.0", "@babel/core@^7.11.0", "@babel/core@^7.12.0", "@babel/core@^7.12.3", "@babel/core@^7.13.0", "@babel/core@^7.14.6", "@babel/core@^7.16.0", "@babel/core@^7.4.0 || ^8.0.0-0 <8.0.0", "@babel/core@^7.7.2", "@babel/core@^7.8.0":
```

Comment:
all of this other stuff needed to be resolved I in order to install the new docs package, does that sound right @kumar303 ?

Reply:
IMO it seems weird that non-`generate-doc` dependencies would have changed in the `yarn.lock` file. Would that be because `yarn -i` needed to have been run in a previous pr and/or `package.json` change?

Reply:
Maybe? 

in this case I actually ran 

`yarn add @shopify/generate-docs@0.16.4`

To avoid updating any other dependencies, but that was the result, maybe I'm doing something wrong there ? wdyt?